packages/@react-stately/actiongroup/src/useActionGroupState.ts

```diff
@@ -29,6 +29,13 @@ export function useActionGroupState<T extends object>(props: ActionGroupProps<T>
     return new TreeCollection(nodes);
   }, [builder, props, disabledKeys]);
 
+  // Reset focused key if that item is deleted from the collection.
```

Comment:
should we make this a hook for collections? given that it was just copied into 4 files

Reply:
I actually tried that but it was a weird hook. Couldn't figure out what to call it!

Reply:
I don't know if I would go for a whole hook just for this useEffect. We will probably have to refine the reset of focus in the near future so we can revisit it then.